recipes/featuretools-tsfresh-primitives/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "featuretools-tsfresh-primitives" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 91e6aeb57df9f23cb01c5ce9aa8e2ddb1c27c5815a8ca308258b9c597898a8c5
+
+build:
+  noarch: python 
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - tsfresh >=0.15.0
+    - pandas >=1.3.0,<2.0.0
+    - featuretools >=1.0.0 
+    - statsmodels <=0.12.2 
+
+test:
+  imports:
+    - featuretools_tsfresh_primitives
+  requires:
```

Comment:
Are the packages outside of `pytest` needed to actually run the tests? I see that they are in the `test_requires`, but they don't look like you actually need them to run the `pytest` command.

Reply:
Nope--removed! Thanks for pointing out.